### PR TITLE
Client

### PR DESCRIPTION
**Pull request by Muszchrom**
testning n stuff n testing

### CODE REVIEW

**Review by Muszchrom — COMMENTED** [review]
Hello world

**Review by MuszchromsClone — APPROVED** [review]
Hello world again i guess

**Review by MuszchromsClone — APPROVED** [review]
...

File: _redirects.txt
Comment (MuszchromsClone) [inline _redirects.txt:1]: Lorem

File: client/_redirects.txt
Comment (MuszchromsClone) [inline client/_redirects.txt:1]: ipsum

**Review by MuszchromsClone — APPROVED** [review]
qwe

File: client/src/App.css
Comment (MuszchromsClone) [inline client/src/App.css:1]: Do i really

File: client/src/App.tsx
Comment (MuszchromsClone) [inline client/src/App.tsx:1]: nneed to

File: client/src/components/Overlay.tsx
Comment (MuszchromsClone) [inline client/src/components/Overlay.tsx:1]: cooment

File: client/src/components/boardAssets/King.tsx
Comment (MuszchromsClone) [inline client/src/components/boardAssets/King.tsx:1]: every

File: client/src/components/boardAssets/Piece.tsx
Comment (MuszchromsClone) [inline client/src/components/boardAssets/Piece.tsx:1]: single

File: client/src/routes/Board.tsx
Comment (MuszchromsClone) [inline client/src/routes/Board.tsx:1]: file

File: client/src/routes/Form.tsx
Comment (MuszchromsClone) [inline client/src/routes/Form.tsx:1]: changed?

**Review by MuszchromsClone — APPROVED** [review]
uisjbcugcug

**Review by Muszchrom — COMMENTED** [review]
let's see what it does

**Review by Muszchrom — COMMENTED** [review]
1234dq

**Review by MuszchromsClone — APPROVED** [review]

**Review by MuszchromsClone — APPROVED** [review]
BROOOO

**Review by MuszchromsClone — APPROVED** [review]
Please work

**Review by MuszchromsClone — APPROVED** [review]
PLEASE WORK

**Review by MuszchromsClone — APPROVED** [review]
rqwerqwrqwe

File: client/src/App.css [hunk 1]
```diff
@@ -5,38 +5,14 @@
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
```
Comment (MuszchromsClone) [inline client/src/App.css:13]: Bro what thef uck

File: client/src/routes/Form.tsx [hunk 2]
```diff
@@ -11,7 +11,12 @@ import { useNavigate } from 'react-router-dom'
 
 import Input from "../components/Input"
 
-export default function Form() {
+interface FormProps {
+  uname: string
+  setUname: (str: string) => void
+}
+
```
Comment (MuszchromsClone) [inline client/src/routes/Form.tsx:18]: qwperjtqnpfea adfq

**Review by MuszchromsClone — COMMENTED** [review]
Please go thru

**Pull request merged**